pycbc/workflow/minifollowups.py

```diff
@@ -340,10 +344,125 @@ class PlotQScanExecutable(PlotExecutable):
+        that *do not* match strings in this list.
+    require : list (optional, default=None)
+        If given, then when considering which subsections in the ini file to
+        parse for options to add to single_template_plot, only use subsections
+        matching strings in this list.
+    tags : list (optional, default=None)
+        The tags to use for this job.
+    store_file : boolean (optional, default=False)
+        Keep the output files of this job.
+    use_mean_time : boolean (optional, default=False)
+        Use the mean time as the center time for all ifos
+    use_exact_inj_params : boolean (optional, default=False)
+        If True do not use masses and spins listed in the params dictionary
+        but instead use the injection closest to the filter time as a template.
+
+    Returns
+    --------
+    output_files : workflow.FileList
+        The list of workflow.Files created in this function.
+    """
+    tags = [] if tags is None else tags
+    makedir(out_dir)
+    name = 'single_template'
+    secs = requirestr(workflow.cp.get_subsections(name), require)
+    secs = excludestr(secs, exclude)
+    secs = excludestr(secs, workflow.ifo_combinations)
+    # Reanalyze the time around the trigger in each detector
+    curr_exe = SingleTemplateExecutable(workflow.cp, 'single_template',
+                                        ifos=[ifo], out_dir=out_dir,
+                                        tags=tags)
+    start = int(params[ifo + '_end_time'])
+    end = start + 1
+    cseg = segments.segment([start, end])
+    node = curr_exe.create_node(valid_seg=cseg)
+
+    if use_exact_inj_params:
+        node.add_opt('--use-params-of-closest-injection')
+    else:
+        node.add_opt('--mass1', "%.6f" % params['mass1'])
+        node.add_opt('--mass2', "%.6f" % params['mass2'])
+        node.add_opt('--spin1z',"%.6f" % params['spin1z'])
+        node.add_opt('--spin2z',"%.6f" % params['spin2z'])
+        node.add_opt('--template-start-frequency',
+                     "%.6f" % params['f_lower'])
+        # Is this precessing?
+        if 'u_vals' in params or 'u_vals_%s' % ifo in params:
+            node.add_opt('--spin1x',"%.6f" % params['spin1x'])
+            node.add_opt('--spin1y',"%.6f" % params['spin1y'])
+            node.add_opt('--spin2x',"%.6f" % params['spin2x'])
+            node.add_opt('--spin2y',"%.6f" % params['spin2y'])
+            node.add_opt('--inclination',"%.6f" % params['inclination'])
+            try:
+                node.add_opt('--u-val',"%.6f" % params['u_vals'])
+            except:
+                node.add_opt('--u-val',
+                             "%.6f" % params['u_vals_%s' % ifo])
+
+    # str(numpy.float64) restricts to 2d.p. BE CAREFUL WITH THIS!!!
+    if params[ifo + '_end_time'] > 0 and not use_mean_time:
+        str_trig_time = f"{params[ifo + '_end_time']:.6f}"
+    else:
+        str_trig_time = f"{params['mean_time']:.6f}"
+    node.add_opt('--trigger-time', str_trig_time)
```

Comment:
I would do this like so:
```suggestion
    if params[ifo + '_end_time'] > 0 and not use_mean_time:
        trig_time = params[ifo + '_end_time']
    else:
        trig_time = params['mean_time']
    node.add_opt('--trigger-time', f'{trig_time:.6f}')
```
so there is only one string conversion.